Sprint-2/debug/recipe.js

```diff
@@ -12,4 +12,6 @@ const recipe = {
 
 console.log(`${recipe.title} serves ${recipe.serves}
   ingredients:
-${recipe}`);
+${recipe.ingredients.join("\n")}`);
```

Comment:
I think you need one `new line` before the list of ingredients, otherwise it will look like:

"... ingredients: olive oil
tomatoes
..."

I think the idea is to have:

"... ingredients:
olive oil
tomatoes
..."